docs/quickstart.rst

```diff
@@ -142,6 +161,20 @@ Open Telemetry
 
 See :ref:`otel` to configure metrics, traces and logging.
 
+Views
+-----
+
+CSRF settings
+"""""""""""""
+
+In order to properly handle the CSRF protection, add the following settings to your Django configuration:
```

Comment:
```suggestion
CSRF failures because of double-clicking the login button are common. Our custom CSRF failure
view detects this, but it requires configuring custom settings:
```